### ISSUE TITLE
Lyrics Quality is NaN

### ISSUE THREAD

**Issue by wimbledon** [XQS6LB3A/lyrics-module#5]
After upgrading from Swift package version 0.10.0 to 0.11.0,
Lyrics Quality is returning NaN.

Specifically, `Lyrics+Quality.swift` following line returns a negative number.
`pow` of a negative base number is `-nan`

```
var quality = 1 - pow((qualityMixBound - artistQuality) * (qualityMixBound - titleQuality) * (qualityMixBound - durationQuality), 0.3333)
```

![image](https://user-images.githubusercontent.com/3291853/125164845-4e1dfc80-e1cf-11eb-8af7-625411316154.png)

**Comment by wimbledon** [comment]
See https://stackoverflow.com/questions/66476503/swift-pow-function-returns-nan

**Comment by ddddxxx** [comment]
released v0.11.1

**Comment by wimbledon** [comment]
@ddddxxx, I'm still getting both `Nan` and `-Inf` values in 0.11.1 version.
